buildroot-external/rootfs-overlay/etc/NetworkManager/system-connections/default

```diff
@@ -2,6 +2,8 @@
 id=HassOS default
 uuid=f62bf7c2-e565-49ff-bbfc-a4cf791e6add
 type=802-3-ethernet
+llmnr=2
+mdns=2
```

Comment:
Can we do, so we need at this also on our profile over supervisor

Reply:
I guess we can set this safely in supervisor since it won't affect older releases which don't use systemd-resolved yet.